### ISSUE TITLE
Blank screen when configuring unconfigurable chart axes

### ISSUE THREAD

**Issue by tsmacdonald** [metabase/metabase#14401]
When configuring waterfall data, the axes dropdown is greyed-out since there's only one item to choose, but I can click the gear next to it which opens up a totally blank sidebar.

From here (`/question/:id`)

![image](https://user-images.githubusercontent.com/784417/104634911-d9766500-5666-11eb-89c0-c277e18e6b5c.png)

Click either of the gears and get:

![Uploading image.png…]()

**Comment by mazameli** [comment]
This isn't particular to waterfall charts, but I think you should at least be seeing an input box to change the name of that column/axis. The gear is meant to open up the formatting settings for that column.

Same thing happens here in a bar chart:

![image](https://user-images.githubusercontent.com/2223916/104635606-fa36be80-5656-11eb-84e8-b79c951b3aa4.png)

**Comment by flamber** [comment]
Very related to #11395 - it will show "Column title" customization if using Table visualization.

**Comment by luizarakaki** [comment]
Adding some context to the issue. Two things to solve:
https://www.loom.com/share/57303b1c6f7b4668ac0f800778c188f9

- [ ] It should save the new name if I click outside the box
- [ ] It should be possible to change the column name from other visualization types

**Comment by iethree** [comment]
> Adding some context to the issue. Two things to solve: 
> * [ ]  It should save the new name if I click outside the box
> * [ ]  It should be possible to change the column name from other visualization types

@luizarakaki , when you get a moment could you test the click-outside-the-popover again? I think we may have fixed it along the way since you made this video.

I was not able to reproduce manually or in e2e tests

![save_click_outside](https://github.com/metabase/metabase/assets/30528226/ad2ad6c1-05b5-4316-8df6-6e2e6052ac87)

**Comment by luizarakaki** [comment]
@iethree yeah, can confirm it is saving when I click outside the box

**Comment by cdeweyx** [comment]
Revisiting this issue, I think there are three problems to solve here still:
- Can’t change x-axis dimension column name (clear, should be able to rename on "..." click)
- Can’t change uncharted column names in tooltips (expected behavior unclear)
- We don’t factor in changed column names from table → other viz types ([previous comment on this](https://github.com/metabase/metabase/issues/14401#issuecomment-1673502806))

First two points, while super basic, are feature requests. This issue is around the last point, and is still open. Will require deeper thinking about how these things interact / how this should be implemented.

**Comment by mazameli** [comment]
Yeah, with fresh eyes, I think this issue is identifying an annoyance with ergonomics for changing axis labels, but is incorrect in its proposed fix. This is probably a wont-fix, but then we can open a feature request to make it easier to change an axis' label, e.g. by clicking on it on the chart itself.

**Comment by Tony-metabase** [comment]
@mazameli and @alxnddr  what about the Tooltip name? Would you consider that as a bug then, cause right now even if you change the column name from the Axis tab the name on the tooltip doesn't change:

<img width="1728" height="775" alt="Image" src="https://github.com/user-attachments/assets/14347ad9-eb52-4fb6-8fb7-3982488d1329" />

I think we can close this and open a specific bug for the Tooltip name that doesn't change?

**Comment by mazameli** [comment]
@Tony-metabase, yeah, let's close this and open a different issue for the tooltip. We'd just need to discuss if the x/y axis label should indeed be what the tooltip shows.

**Comment by Tony-metabase** [comment]
Thanks @mazameli!

Closing this in favor of the following https://github.com/metabase/metabase/issues/67778

Adding that the issue is only a problem on Row Chats:

<img width="2258" height="1100" alt="Image" src="https://github.com/user-attachments/assets/a05c5bf9-3d8b-4072-b4d7-e52b2110cce3" />

Bar/Line Charts Tooltip is fine

<img width="3456" height="1588" alt="Image" src="https://github.com/user-attachments/assets/191ca8d6-947a-47cb-a454-46647f5ec2f8" />
